### DISCUSSION TITLE
Multiple Component Namespaces

### DISCUSSION THREAD

**Discussion by carbonvader** [livewire/livewire#4665] [3. Feature ideas]
Hello,

**Need:** My application has seperate namespaces for seperate domains such belove. That's why I had to hack LivewireComponentsFinder in order not to lose Livewire's **auto discovery** feature. 

- App
- - Backend
- - - Http
- - - - Livewire
- - Frontend
- - - Http
- - - - Livewire

**Idea:** It would be so nice to have regiter multiple namespaces in config file or service provider as Blade supports like that. `    Blade::componentNamespace('Nightshade\\Views\\Components', 'nightshade');`

Thanks for this great package.

**Comment by joshhanley** [comment]
@carbonvader yep definitely can see a need for that. There is a similar feature idea already open #4496 🙂

PR's are welcome!
